content/en/database_monitoring/setup_mongodb/amazon_documentdb.md

```diff
@@ -0,0 +1,241 @@
+---
+title: Setting Up Database Monitoring for Amazon DocumentDB
+further_reading:
+    - link: '/integrations/mongo/'
+      tag: 'Documentation'
+      text: 'Basic MongoDB Integration'
+---
+
+Database Monitoring offers comprehensive insights into your DocumentDB databases by providing access to critical metrics, operation samples, explain plans, and replication state changes. To take advantage of Database Monitoring for Amazon DocumentDB, ensure that the Datadog Agent is installed and configured to connect to your DocumentDB instances. This guide outlines the steps to set up Database Monitoring for Amazon DocumentDB.
+
+## Before you begin
+
+Supported Amazon DocumentDB major versions
+: 4.4, 5.0
+
+{{% dbm-mongodb-before-you-begin %}}
```

Comment:
Let's also add a `Supported Amazon DocumentDB cluster types` and indicate elastic cluster is not supported